app/views/home/_team.html.erb

```diff
@@ -5,165 +5,32 @@
  <br>
 
  <div class="row">
-
-   <div class="col s12 m4"> <!-- MEMBRO 01  -->
-     <div class="card hoverable">
-       <div class="card-image waves-effect waves-block waves-light">
-         <%= image_tag("team/farma-01.jpg", class: "activator") %>
-       </div>
-
-       <div class="card-reveal">
-         <span class="card-title white-text">
-           <h5>Diego Marczal</h5>
-           <h6>Idealizador da FARMA <br /> Desenvolvimento Web</h6>
-         </span>
-         <p>
-           Professor Adjunto da Universidade Tecnológica Federal do Paraná -
-           Câmpus Guarapuava
-         </p>
-         <a href="http://diego.marczal.com" class="right"><b>SAIBA MAIS</b></a>
-       </div>
-     </div>
-   </div>
-
-   <div class="col s12 m4"> <!-- MEMBRO 02  -->
-     <div class="card hoverable">
-       <div class="card-image waves-effect waves-block waves-light">
-         <%= image_tag("team/farma-02.jpg", class: "activator") %>
-       </div>
-       <div class="card-reveal">
-         <span class="card-title white-text">
-           <h5>Eleandro Maschio</h5>
-           <h6>Inteligência Artificial aplicada à Educação</h6>
-         </span>
-         <p>
-           Professor Adjunto da Universidade Tecnológica Federal do Paraná -
-           Câmpus Guarapuava
-         </p>
-         <a href="http://lattes.cnpq.br/5642072550423634" class="right"><b>SAIBA MAIS</b></a>
-       </div>
-     </div>
-   </div>
-
-   <div class="col s12 m4"> <!-- MEMBRO 03  -->
-     <div class="card hoverable">
-       <div class="card-image waves-effect waves-block waves-light">
-         <%= image_tag("team/farma-03.jpg", class: "activator") %>
-       </div>
-
-       <div class="card-reveal">
-         <span class="card-title white-text">
-           <h5>Alex S. de Castilho</h5>
-           <h6>Matemática</h6>
-         </span>
-         <p>
-           Professor Assistente da Universidade Tecnológica Federal do Paraná -
-           Câmpus Guarapuava
-         </p>
-         <a href="http://lattes.cnpq.br/0971381050120580" class="right"><b>SAIBA MAIS</b></a>
-       </div>
-     </div>
-   </div>
-
-   <div class="col s12 m4"> <!-- MEMBRO 04  -->
+  <% @developers.each do |dev| %>
```

Comment:
Its better to [render a collection](http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections)